src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -329,6 +330,23 @@ private void RunAssembly(string assemblyPath, TestFilter filter)
             }
         }
 
+
+        private void GenerateTestOutput(XmlNode testResults, string assemblypath)
+        {
+            if (!Settings.UseTestOutputXml)
+                return;
+#if NETCOREAPP1_0
+#else
```

Comment:
`#if !NETCOREAPP1_0`?

Reply:
No, the else block It should only be run if .net framework, not .net core.  Missing a #ifndef directive

Reply:
@OsirisTerje `#if !NETCOREAPP1_0` is that directive. I'm confused because I don't see a difference between `#if !NETCOREAPP1_0` and what you have, except that it's shorter.

Reply:
You're right.  Will fix.